extensions/iceberg/src/main/java/io/deephaven/iceberg/util/IcebergCatalogAdapter.java

```diff
@@ -147,31 +140,37 @@ public List<Long> listTableSnapshots(@NotNull final TableIdentifier tableIdentif
      * Read the latest static snapshot of a table from the Iceberg catalog.
      *
      * @param tableIdentifier The table identifier to load
+     * @param instructions The instructions for customizations while reading
      * @return The loaded table
      */
     @SuppressWarnings("unused")
-    public Table snapshotTable(@NotNull final TableIdentifier tableIdentifier) {
-        return readTableInternal(tableIdentifier, -1, false);
+    public Table snapshotTable(
```

Comment:
Maybe we call these `readTable` in order to distinguish from `snapshotTable()` as a `Table` of `Snapshots`.